lib/budo.js

```diff
@@ -2,54 +2,89 @@ const path = require('path')
 
 const budo = require('budo')
 
-const transformCss = require('./css-transform')
 const logger = require('./logger')
 const transformJs = require('./js-transform')
 
-module.exports = function ({ config, env, files, flyle, instrument, proxy }) {
+module.exports = function ({ config, env, files, flyle, instrument, minify, outdir, proxy, watch }) {
+
+  logger.log('Starting budo sever...')
+
+  const budoFiles = []
+  files.map(file => {
```

Comment:
This is a minor thing, since this works as-is, but I'd probably prefer `forEach` here rather than `map` since the return value of `map` is being ignored.

Reply:
This syntax was in the original source, but moved slightly.

Reply:
Sorry, I think I missed the deletion block in my pass, I was toggling between the two PRs and I think I skipped over it. Up to you whether you want to spend any more time on cleanup.